circus-train-core/src/test/java/com/hotels/bdp/circustrain/core/metastore/ThriftHiveMetaStoreClientFactoryTest.java

```diff
@@ -15,21 +15,19 @@
  */
 package com.hotels.bdp.circustrain.core.metastore;
 
-import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
-import java.util.Arrays;
-import java.util.List;
-
-import org.apache.hadoop.hive.metastore.api.FieldSchema;
 import org.junit.Test;
 
-public class FieldSchemaUtilsTest {
+public class ThriftHiveMetaStoreClientFactoryTest {
```

Comment:
Not the end of the world but this is a bit confusing. I'm guessing you took FieldSchemaUtilsTest and renamed it to this class which is why git is showing it as a diff instead of as new.